spec/02-integration/05-proxy/11-error_default_type_spec.lua

```diff
@@ -42,6 +42,15 @@ describe("error_default_type", function()
 
   describe("request `Accept` is missing", function()
     describe("(default)", function()
+      setup(function()
+        assert(helpers.kong_exec("reload", {
+          prefix             = helpers.test_conf.prefix,
+          nginx_conf         = "spec/fixtures/custom_nginx.template",
+        }))
+
+        ngx.sleep(RELOAD_DELAY)
```

Comment:
Already done in  #2827. I think I might have seen the test fail nonetheless, but I'm not 100% sure.